examples/render/quad_render/main.rs

```diff
@@ -231,14 +228,13 @@ fn main() {
 
     {
         let mut data = context.mut_device()
-            .acquire_mapping_writer(&image_upload_buffer, 0..upload_size)
+            .write_mapping(&image_upload_buffer, 0..upload_size)
```

Comment:
I believe having the whole block in `if let Ok(mapping) = device.write_mapping(...)` looks much nicer

Reply:
It won't crash if there is an error though, but maybe that's what you want ? Otherwise I don't think the block is necessary.

Reply:
yeah, it wouldn't, so it's not much better, just looks somewhat nicer